### PR TITLE
Added amplify for graphql functionality

### PR DESCRIPTION
**Pull request by AlecPierce**

### CODE REVIEW

File: web-app/src/models/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { ModelInit, MutableModel, PersistentModelConstructor } from "@aws-amplify/datastore";
+
```
Comment (AaronDingman) [inline web-app/src/models/index.d.ts:2]: Too many new lines.

File: web-app/src/models/schema.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                    "properties": {}
+                },
+                {
+                    "type": "auth",
+                    "properties": {
+                        "rules": [
+                            {
+                                "allow": "public",
+                                "operations": [
+                                    "create",
```
Comment (AaronDingman) [inline web-app/src/models/schema.js:49]: Do you need all of these available or just a create / read?

File: web-app/src/models/schema.js [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+                    }
+                },
+                {
+                    "type": "auth",
+                    "properties": {
+                        "rules": [
+                            {
+                                "allow": "public",
+                                "operations": [
+                                    "create",
```
Comment (AaronDingman) [inline web-app/src/models/schema.js:101]: Same here.

File: web-app/src/models/index.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// @ts-check
+import { initSchema } from '@aws-amplify/datastore';
+import { schema } from './schema';
+
```
Comment (AaronDingman) [inline web-app/src/models/index.js:4]: Too many new lines.

**Review by AaronDingman — CHANGES_REQUESTED** [review]
Looks good, some small changes, and a few questions.

File: web-app/src/models/schema.js (hunk 3 above)
Comment (AlecPierce) [reply web-app/src/models/schema.js:101]: Unsure about this, but it was auto-generated so will remove later if some aren't used.

File: web-app/src/models/index.d.ts (hunk 1 above)
Comment (AlecPierce) [reply web-app/src/models/index.d.ts:2]: Mmmk

File: web-app/src/models/index.js (hunk 4 above)
Comment (AlecPierce) [reply web-app/src/models/index.js:4]: Mmmk

**Review by AaronDingman — APPROVED** [review]
Looks good!

**Pull request closed without merging**